qtpy/tests/test_uic.py

```diff
@@ -56,8 +58,9 @@ def test_load_ui(qtbot):
 
 
 @pytest.mark.skipif(
-    PYSIDE2 or PYSIDE6,
-    reason="PySide2uic not consistently installed across platforms/versions")
+    PYSIDE6 and using_conda() and parse(PYSIDE_VERSION) <= parse("6.4")
```

Comment:
I think the tests are failing with Python 3.11 since there we use PySide6 `6.4.3` which is not <= `6.4`. On the other hand with Python 3.7 we test using PySide6 `6.4.0` so that skips the test. So something like this should be enough for the test to pass:

```suggestion
    PYSIDE6 and using_conda() and parse(PYSIDE_VERSION) <= parse("6.4.3")
```

Reply:
Ah thanks, stupid mistake. I tried a more general solution which is compatible with possible future releases of 6.4.*.